src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,29 @@
+appenders = console
```

Comment:
Rename this file to log4j2-json.properties. And add to the documentation that if log in json format needed then specify this file via env variable